src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration.Performance/ConfigurationClientBenchmark.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.Pipeline;
+using BenchmarkDotNet.Attributes;
+
+namespace Azure.ApplicationModel.Configuration.Performance
+{
+    [MemoryDiagnoser]
+    public class ConfigurationClientBenchmark
+    {
+        private static readonly string ConnectionString = "Endpoint=https://contoso.appconfig.io;Id=b1d9b31;Secret=aabbccdd";
```

Comment:
You may want to consider a comment that clarifies that this should not be a real connection string to an actual Azure resource.... just to avoid any confusion for those maintaining in the future.

Reply:
Renamed the property.